src/Entities/Misc/AttachedWallBooster.cs

```diff
@@ -45,6 +45,13 @@ public AttachedWallBooster(EntityData data, Vector2 offset)
         {
             Remove(Get<CoreModeListener>());
         }
+
+        string spriteField = data.String("sprite", "");
+        if (spriteField != "")
```

Comment:
use `string.IsNullOrEmpty`
```suggestion
        if (!string.IsNullOrEmpty(spriteField))
```